x509/src/ext/mod.rs

```diff
@@ -0,0 +1,45 @@
+//! Standardized X.509 Certificate Extensions
```

Comment:
Regarding a `mod.rs`, these files were effectively soft deprecated by [RFC2126: path clarity](https://rust-lang.github.io/rfcs/2126-path-clarity.html) and the 2018 edition.

Stylistically across the RustCrypto project we don't use `mod.rs` files.

Reply:
@tarcieri I have no problem complying with this project's convention.

However, I don't think the claim that this is deprecated by Rust is actually correct. The RFC explicitly states `The use of mod.rs continues to be allowed without any deprecation...`.

There's an "old" way and a "new" way. The new way was created to improve the editor experience. But now the editors have better support for the old way than the new way. So, IMHO, the old way is actually the better choice.

But again, I really don't care all that much and I'm happy to follow this project's conventions.

Reply:
Everything else aside, I think it’s a bad idea to use a mixture of styles, and across all of the RustCrypto we use the “new” way by convention

Reply:
This is now fixed.